x-pack/platform/packages/shared/onechat/onechat-common/tools/tool_result.ts

```diff
@@ -36,9 +54,13 @@ export interface ResourceResult {
 export interface TabularDataResult {
   type: ToolResultType.tabularData;
   data: {
-    columns: EsqlEsqlColumnInfo[];
-    values: FieldValue[][];
+    esqlQuery: string;
+    esqlResult: {
+      columns: EsqlEsqlColumnInfo[];
+      values: FieldValue[][];
+    };
```

Comment:
This was meant to be generic tabular data, so renaming those to `esqlXXX` kinda goes against that idea.

Reply:
Fixed